flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingMetricCollector.java

```diff
@@ -223,16 +220,26 @@ private void updateKafkaSourceMaxParallelisms(
      * Given a map of collected Flink vertex metrics we compute the scaling metrics for each job
      * vertex.
      *
-     * @param collectedMetrics Collected metrics for all job vertices.
+     * @param flinkMetrics Collected metrics for all job vertices.
      * @return Computed scaling metrics for all job vertices.
      */
     private CollectedMetrics convertToScalingMetrics(
             ResourceID resourceID,
-            Map<JobVertexID, Map<FlinkMetric, AggregatedMetric>> collectedMetrics,
+            Map<JobVertexID, Map<FlinkMetric, AggregatedMetric>> flinkMetrics,
```

Comment:
```suggestion
            Map<JobVertexID, Map<FlinkMetric, AggregatedMetric>> collectedMetrics,
```

Together with my comment above.